src/aap_eda/api/serializers/auth.py

```diff
@@ -114,19 +131,18 @@ class RoleDetailSerializer(serializers.Serializer):
 
     permissions = PermissionRefSerializer(read_only=True, many=True)
 
-    created_at = serializers.DateTimeField(
-        required=True,
-        help_text="The created_at timestamp of the role",
-    )
+    created_at = serializers.SerializerMethodField()
+    modified_at = serializers.SerializerMethodField()
 
-    modified_at = serializers.DateTimeField(
-        required=True,
-        help_text="The modified_at timestamp of the role",
-    )
+    def get_created_at(self, obj):
+        return obj.created
+
+    def get_modified_at(self, obj):
+        return obj.modified
```

Comment:
IMO It's crucial to document the purpose of these methods and its role in satisfying the DAB interface requirements. A detailed docstring would help future developers understand why this method exists and prevent accidental removal if its usage isn't immediately obvious.